pallets/ddc-staking/src/weights.rs

```diff
@@ -35,153 +35,306 @@ pub trait WeightInfo {
 			.saturating_add(RocksDbWeight::get().reads(5_u64))
 			.saturating_add(RocksDbWeight::get().writes(3_u64))
 	}
 	// Storage: DdcClusters Clusters (r:1 w:0)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Ledger (r:1 w:0)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcClusters ClustersGovParams (r:1 w:0)
+	// Proof Skipped: DdcClusters ClustersGovParams (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Providers (r:1 w:0)
+	// Proof Skipped: DdcStaking Providers (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Storages (r:1 w:1)
+	// Proof Skipped: DdcStaking Storages (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking LeavingStorages (r:1 w:0)
+	// Proof Skipped: DdcStaking LeavingStorages (max_values: None, max_size: None, mode: Measured)
 	fn store() -> Weight {
-		Weight::from_parts(28_000_000_u64, 0)
+		Weight::from_parts(471_000_000_u64, 0)
 			.saturating_add(RocksDbWeight::get().reads(6_u64))
 			.saturating_add(RocksDbWeight::get().writes(1_u64))
 	}
 	// Storage: DdcStaking Ledger (r:1 w:1)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Storages (r:1 w:1)
+	// Proof Skipped: DdcStaking Storages (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcClusters ClustersGovParams (r:1 w:0)
+	// Proof Skipped: DdcClusters ClustersGovParams (max_values: None, max_size: None, mode: Measured)
 	fn chill() -> Weight {
-		Weight::from_parts(28_000_000_u64, 0)
+		Weight::from_parts(394_000_000_u64, 0)
 			.saturating_add(RocksDbWeight::get().reads(3_u64))
 			.saturating_add(RocksDbWeight::get().writes(2_u64))
 	}
 	// Storage: DdcStaking Bonded (r:1 w:1)
+	// Proof Skipped: DdcStaking Bonded (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Ledger (r:2 w:2)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	fn set_controller() -> Weight {
-		Weight::from_parts(14_000_000_u64, 0)
+		Weight::from_parts(279_000_000_u64, 0)
 			.saturating_add(RocksDbWeight::get().reads(3_u64))
 			.saturating_add(RocksDbWeight::get().writes(3_u64))
 	}
 	// Storage: DdcStaking Nodes (r:1 w:2)
+	// Proof Skipped: DdcStaking Nodes (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Providers (r:1 w:1)
+	// Proof Skipped: DdcStaking Providers (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Storages (r:1 w:0)
+	// Proof Skipped: DdcStaking Storages (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking LeavingStorages (r:1 w:0)
+	// Proof Skipped: DdcStaking LeavingStorages (max_values: None, max_size: None, mode: Measured)
 	fn set_node() -> Weight {
-		Weight::from_parts(14_000_000_u64, 0)
+		Weight::from_parts(296_000_000_u64, 0)
```

Comment:
The weights were taken from Rakan's PR and was executed on a dedicated benchmarking machine